recipes/omniorbpy/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set version = "4.2.3" %}
+
+package:
+  name: omniorbpy
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/omniorb/files/omniORBpy/omniORBpy-{{ version }}/omniORBpy-{{ version }}.tar.bz2
+  sha256: 5c601888e57c7664324357a1be50f2739c468057b46fba29821a25069fc0aee5
+  patches:
+    - patch-configure.diff
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - autoconf
+    - make
+  host:
+    - python
+    - omniorb
+    - openssl
+  run:
+    - python
+    - omniorb
+
+test:
+  source_files:
+    - python3/omniidl_be
```

I assume this is needed?
```suggestion
    - python/omniidl_be  # [py2k]
    - python3/omniidl_be  # [py3k]
```
 (and similar for the commands section)